gems/aws-sdk-core/lib/aws-sdk-core/plugins/telemetry.rb

```diff
@@ -0,0 +1,73 @@
+# frozen_string_literal: true
+
+module Aws
+  module Plugins
+    # @api private
+    class Telemetry < Seahorse::Client::Plugin
+      option(
+        :telemetry_provider,
+        default: Aws::Telemetry::NoOpTelemetryProvider,
+        doc_type: Aws::Telemetry::TelemetryProviderBase,
+        rbs_type: Aws::Telemetry::TelemetryProviderBase,
```

Comment:
Does rbs_type default to doc_type?

Reply:
Far as I looked, I don't think so for V3 - I didn't see anything special done in [PluginOption](https://github.com/aws/aws-sdk-ruby/blob/version-3/gems/aws-sdk-core/lib/seahorse/client/plugin.rb#L110) class when it comes to `rbs_type` so I will be leaving this as it is.